framework/include/problems/FEProblemBase.h

```diff
@@ -2114,7 +2110,14 @@ class FEProblemBase : public SubProblem, public Restartable
   void createTagSolutions();
 
   MooseMesh & _mesh;
-  EquationSystems _eq;
+
+private:
+  /// The restartable name used for the restartable EquationSystems
+  static const std::string equation_systems_restartable_name;
+  /// The EquationSystems object, wrapped for restart
+  Restartable::ManagedValue<RestartableEquationSystems> _req;
+
+protected:
```

Comment:
I know... this one is kind of important. It's a "managed" value because it needs to get destructed at the proper time, and there's other protected vars that will scream and die otherwise. Hence, I put it as far up as I could.